WalletWasabi/WabiSabi/Client/CoinJoin/Manager/CoinJoinManager.cs

```diff
@@ -527,7 +528,15 @@ private async Task HandleCoinJoinFinalizationAsync(CoinJoinTracker finishedCoinJ
 		catch (CoinJoinClientException clientException)
 		{
 			cjClientException = clientException;
-			Logger.LogDebug(clientException);
+			if (cjClientException.CoinjoinError is CoinjoinError.CoordinatorLiedAboutInputs)
+			{
+				Logger.LogError(cjClientException);
+				forceStop = true;
```

Comment:
Great idea. We should forecefully stop and not try CJing again with a lying coordinator.